test/driver.js

```diff
@@ -431,6 +431,9 @@ class Driver {
       task.stats = { times: [] };
       task.enableXfa = task.enableXfa === true;
 
+      // just in case scale is not set, setting to default=1
```

Comment:
In this case, the comment doesn't really add all that much value (compared to simply reading the code); hence I'd suggest removing this one.

(For future reference, this type of comments should be a *complete* sentence. That means beginning with an upper-case letter and ending with a period.)